Cargo.toml

```diff
@@ -21,15 +23,17 @@ default = [
   'chacha',
   'ecies',
   'curve25519',
+  'certificate',
   'ser',
   'blake',
   'sha3',
   'rfc5649',
 ]
 aes = ['aead', 'aes-gcm']
 chacha = ['aead', 'chacha20poly1305', 'chacha20']
+certificate = ['curve25519', 'x509-cert', 'uuid']
 ecies = ['aead', 'crypto_box']
-curve25519 = ['curve25519-dalek', 'ed25519-dalek', 'signature']
+curve25519 = ['curve25519-dalek', 'ed25519-dalek', 'signature', "pkcs8"]
```

Comment:
Can this be linked to `certificate` instead? I don't really want to have to compile it when working on CoverCrypt.

Reply:
right